### ISSUE TITLE
Improve responsability of files

### ISSUE THREAD

**Issue by vmarcosp** [iagolaguna/vscodefy-server#1]
I think that the file [routes/index.js](https://github.com/iagolaguna/vscodefy-server/blob/master/src/routes/index.js) has many responsibilities. It's inside of "routes" but it has all the endpoints and its implementations. I suggest separate in some files and folders like [this project](https://github.com/blumenaujs/slack-invite-server/tree/master/src). What u think @iagolaguna?

**Comment by iagolaguna** [comment]
i agree with you @vmarcosp, i already thought about this and i didn't have time to do this before.

i will do this issue today or you want do that?
